passport_expiration/models/hr.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2013 Savoir-faire Linux
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = "hr.employee"
+
+    passport_expiration_date = fields.Date(
+        "Expiration Date", help="Expiration date of the passport."
+    )
```

Comment:
```suggestion
    passport_expiration_date = fields.Date(
        "Expiration Date", help="Expiration date of the passport.", groups="hr.group_hr_user"
    )
```

Reply:
We need to add `groups="hr.group_hr_user"` to avoid getting error:
```
ValueError: Invalid field 'emergency_contact_2' on model 'hr.employee.public'
```
here https://github.com/odoo/odoo/blob/f04ea20e8b0622bf15ed4fb9f35d33e108f5e238/addons/hr/models/hr_employee.py#L166-L170
More context about it here https://github.com/odoo/odoo/blob/f04ea20e8b0622bf15ed4fb9f35d33e108f5e238/addons/hr/models/hr_employee.py#L22-L27